.changeset/plenty-cars-boil.md

```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Change to measured characteristic: borderSize 1px -> 2px; borderStyle states added (active, complete, incomplete); foreground and borderColor tokens; disabled background token
```

Comment:
Could we expand on the changes to measured characteristic. Potentially in a similar format to: https://github.com/jpmorganchase/uitk/pull/589/files#diff-e88587ddac519bdb65910b0a821c5da1396994585d5f00905e46f1441054695f

Reply:
@joshwooding 
https://github.com/jpmorganchase/uitk/pull/532/commits/ef396742436f31a9f248bc9489712379dcbddd59

Reply:
Perfect :)